.devcontainer/devcontainer.json

```diff
@@ -18,6 +18,10 @@
 		"stackbuild.bazel-stack-vscode-cc",
 		"augustocdias.tasks-shell-input",
 		"ryuta46.multi-command",
+		"zxh404.vscode-proto3",
```

Comment:
Could we get a quick description of each of these plugins and what they are used for / enabling / required for in the PR? It can be hard to prune plugins down the road without some understanding.  E.g. my first question was why the proto3 for a python enablement PR (is this general proto3 support? Does an existing proto-* plugin not exist - etc).

Reply:
Of course, added a description. And I deleted the proto extension, since it is indeed not necessary at this stage. We can pick it up at a later stage again, when introducing generation of proto python files for example.